OneOneOne/OneOneOne/tests/test_message_handler.py

```diff
@@ -0,0 +1,154 @@
+import unittest
+from pathlib import Path
+from builder.pystache_message_builder import PystacheMessageBuilder
+from utilities.file_utilities import FileUtilities
+from utilities.xml_utilities import XmlUtilities
+from OneOneOne.OneOneOne.message_handler import MessageHandler
+
+from definitions import ROOT_DIR
+
+
+class MessageHandlerTest(unittest.TestCase):
+    expectedXmlFileDir = Path(ROOT_DIR) / 'OneOneOne' / 'tests' / 'expected_output_xmls'
+    inputXmlFileDir = Path(ROOT_DIR) / 'OneOneOne' / 'tests' / 'input_xmls'
```

Comment:
Your input templates all seem fairly similar too. Could you have a single template that you customise to produce the failure scenario, rather than multiples?

Reply:
Cut down the templates to just the one